packages/compass-e2e-tests/helpers/commands/set-value-visible.ts

```diff
@@ -5,7 +5,18 @@ export async function setValueVisible(
   selector: string,
   value: string
 ): Promise<void> {
-  const element = await browser.$(selector);
-  await element.waitForDisplayed();
-  await element.setValue(value);
+  await browser.waitUntil(
+    async () => {
+      const element = await browser.$(selector);
+      await element.waitForDisplayed();
+      await element.clearValue();
+      await element.setValue(value);
+      const actualValue = await element.getValue();
+      if (actualValue !== value) {
+        console.log(actualValue, '!==', value);
```

Comment:
Trying to debug this. Some race condition where sometimes it appends the connection string on at the end rather than clearing it first. Just leaving this console.log in to keep an eye on it.